.github/workflows/release.yml

```
@@ -41,10 +41,10 @@ env:
 jobs:
   version:
     name: Resolve Version
-    if: github.event_name == 'workflow_dispatch'
     uses: ./.github/workflows/version.yml
     with:
       version: ${{ inputs.version }}
+      is-release: ${{ github.event_name == 'workflow_dispatch' }}
```

actionlint (1.7.12)

[error] input "is-release" is typed as string by reusable workflow "./.github/workflows/version.yml". bool value cannot be assigned

(expression)